src/App.js

```diff
@@ -56,6 +57,178 @@ class App extends Component {
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 3,
+    label: 3,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 4,
+    label: 4,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 5,
+    label: 5,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 6,
+    label: 6,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 7,
+    label: 7,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 8,
+    label: 8,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 9,
+    label: 9,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 10,
+    label: 10,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 11,
+    label: 11,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 12,
+    label: 12,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 13,
+    label: 13,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 14,
+    label: 14,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 15,
+    label: 15,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 16,
+    label: 16,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 17,
+    label: 17,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 18,
+    label: 18,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 19,
+    label: 19,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 20,
+    label: 20,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 21,
+    label: 21,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 22,
+    label: 22,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 23,
+    label: 23,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 24,
+    label: 24,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 25,
+    label: 25,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 26,
+    label: 26,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 27,
+    label: 27,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  },{
+    value: 28,
+    label: 28,
+    color: '#5243AA',
+    isFixed: false,
+    isDisabled: false  
+  }];
+  selectedChapters=[];
+  FomatedSelectedChapters=[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,10,21,22,23,24,25,26,27,28];
```

Comment:
Could these be moved to a constants file? Otherwise it does look good. Hoping I can run it tonight and merge it around 9pm

Reply:
yeah, I can move it to a const file but the long term solution should be getting this from the API.